app/main.py

```diff
@@ -1,3 +1,57 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int or float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km!r})"
+
+    def __add__(self, other: int or "Distance") -> "Distance":
+        if isinstance(other, (int, float)):
+            return Distance(
+                self.km + other
+            )
```

Comment:
There is no need to place `self.km + other` on a separate line, avoid line breaking in such cases